.github/workflows/build.yml

```diff
@@ -86,7 +86,9 @@ jobs:
           GOOS: ${{ matrix.goos }}
           GOARCH: ${{ matrix.goarch }}
           LDFLAGS: ${{ needs.get-product-version.outputs.ldflags }}
+          CGO_ENABLED: "0"
         run: |
+          go env
```

Comment:
Is the idea to leave this `go env` cmd in here; or, was it for testing?

Reply:
I was going to keep it for future reference.

Actually, the [previous step](https://github.com/hashicorp/consul-ecs/runs/6599745996?check_suite_focus=true#step:3:16) installs Go and runs `go env` which is how I was able to check CGO was enabled. But I didn't see how to set CGO_ENABLED=0 persistently in the `actions/setup-go@v2` Github action, so I did it as part of the build and also run `go env` again 🤷